docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
     gateway:
         ports:
             - 8080:8080
-        image: openfaas/gateway:0.13.6
+        image: openfaas/gateway:0.13.7-rc2
```

Comment:
You should be able to remove the `basic_auth: "${BASIC_AUTH:-true}"` env variable now, right?

Reply:
I will check